XamarinCommunityToolkitSample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj

```diff
@@ -117,4 +118,9 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
+  <ProjectExtensions>
+    <VisualStudio>
+      <UserProperties TriggeredFromHotReload="False" />
+    </VisualStudio>
+  </ProjectExtensions>
```

Comment:
Shouldn't be removed?

Reply:
VS always adds this to my csproj. should I remove it?

Reply:
yes, please